src/test/resources/processor_test_files/2111_BigDecimalDoubleConstructor/BigDecimalDoubleConstructor.java.expected

```diff
@@ -26,7 +26,7 @@ public class BigDecimalDoubleConstructor {
     // Aditional tests
     public void foo(String[] args) {
         double d = 1.1;
-        float f = 2.2;
+        float f = 2.2f;
```

Comment:
@slarse Seems good to me.

Just one question: why did you have to change many of the tests like this one?

Reply:
tl;dr: I needed the files to compile.

The Maven launcher test grabs all of the test files from `resources/processor_test_files` and creates a maven project out of them, and then runs a repair. Many of those files did not compile, either because they were syntactically invalid (found a few methods nested in methods and stuff like that), semantically invalid (such as the above, you can't assign a double literal to a float variable), or needlessly contained type references that were not on the classpath.

Reply:
Hmm, I thought so. Seems that we had many noncompilable test files.